packages/frontend-shared/src/components/ResultCounts.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="border rounded flex border-gray-200 h-6 text-gray-700 text-size-14px leading-20px">
+    <div
+      v-for="(result, i) in results"
+      :key="i"
+      class="flex px-2 items-center"
+      :title="result.name"
+      :data-cy="`runResults-${result.name}-count`"
+    >
+      <component
+        :is="result.icon"
+        class="mt-px h-12px mr-1 w-12px"
+        :class="result.class"
+      />
+      <span class="sr-only">{{ result.name }}</span>
+      {{ result.value }}
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+export interface ResultCountsProps {
```

Comment:
I didn't think you can export from `script setup`. I am surprised this works 🤔

Reply:
I guess you can export types